src/deploy/missions/contracts/cross-chain/zk-evm-bridge.ts

```diff
@@ -23,8 +22,8 @@ IZNSContracts
 
   constructor (args : IDeployMissionArgs<
```

Comment:
It looks like this type is only used a single time. It would be simpler to just make the type you need instead of creating a single-use generic type then typing it with the expected values anyways